bundles/automation/org.eclipse.smarthome.automation.module.core/ESH-INF/automation/moduletypes/RuleEnableModuleType.json

```diff
@@ -0,0 +1,26 @@
+{
+	"actions": [
+		{
+			"uid": "RuleEnableAction",
+			"label": "Enables or disables rules",
```

Comment:
This should better be a name of the module, not describing what it does. How about something like "Rule Enablement Action"?